scripts/build_dicts.sh

```diff
@@ -1,6 +1,6 @@
-#!/bin/bash
+#!/bin/sh
 
-go get foosoft.net/projects/yomichan-import/yomichan
+go get github.com/themoeway/yomitan-import/tree/master/yomitan
```

Comment:
I read that `go get` should be replaced with `go install`: https://stackoverflow.com/a/24878851
Is there still a valid reason to use `go get` here by chance?

Reply:
It seems that it has been deprecated, https://go.dev/doc/go-get-install-deprecation so we should switch to go install. for our usage go install is appropriate.

Reply:
Sounds good! As mentioned on discord, I think it would be best to just undo the `bin/sh` change, and then I can take care of the build scripts from there.